examples/hotrod/docker-compose.yml

```diff
@@ -1,7 +1,11 @@
 version: '3.7'
+
+# To run a specific version of Jaeger, use environment variable, e.g.:
+#     JAEGER_VERSION=1.52 docker compose up
```

Comment:
Perhaps we should move this into https://github.com/jaegertracing/jaeger/tree/main/examples/hotrod#run-everything-via-docker-compose (and also update the command)?

Reply:
added to README